.lintrunner.toml

```diff
@@ -224,8 +224,8 @@ exclude_patterns = [
     'third_party/**/*',
     'torch/csrc/api/**',
     'torch/csrc/autograd/generated/**',
-    'torch/csrc/dynamo/*',
     'torch/csrc/distributed/**/*',
+    'torch/csrc/dynamo/eval_frame.h',
```

Comment:
Why ignore this one header?

Reply:
Because clang-tidy reports a strange warning. Let's re-check it.